app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx

```diff
@@ -88,33 +114,49 @@ class DiscoveryMap extends React.Component {
   };
 
   renderMarker = locationInfo => {
-    const { currentTab, previewedLocationId } = this.props;
+    const { currentTab, mapLevel, previewedLocationId } = this.props;
     const { viewport } = this.state;
+
     const id = locationInfo.id;
     const name = locationInfo.name;
     const lat = parseFloat(locationInfo.lat);
     const lng = parseFloat(locationInfo.lng);
+    const geoLevel = locationInfo.geo_level;
+
     const idsField = currentTab === "samples" ? "sample_ids" : "project_ids";
     if (!locationInfo[idsField]) return;
+
     const pointCount = locationInfo[idsField].length;
-    const minSize = 10;
+    // Sizing parameters determined via eyeballing.
+    const minSize = 14;
     // Scale based on the zoom and point count (zoomed-in = higher zoom)
-    // Log1.5 of the count looked nice visually for not getting too large with many points.
+    // Log1.3 of the count looked nice visually for not getting too large with many points.
     const markerSize = Math.max(
-      Math.log(pointCount) / Math.log(1.4) * (get("zoom", viewport) || 3),
+      Math.log(pointCount) / Math.log(1.3) * (get("zoom", viewport) || 3),
```

Comment:
why `Math.log(1.3)` and not `0.26` or `4x`.

Reply:
I was thinking something logarithmic of the pointCount would just keep it from ever getting too large (vs just multiplying), and then Math.log/Math.log is how to set log1.3(pointCount)

Reply:
I guess I would prefer a simple version that would gives a way to customize it... Hard to understand the effect.
Would we ever have different scenario where we need to change this?